frontend/src/Track.elm

```diff
@@ -22,64 +24,54 @@ import PagesComponents.Organization_.Project_.Models.FindPathResult exposing (Fi
 -- all tracking events should be declared here to have a good overview of them
 
 
-openAppCta : String -> TrackEvent
-openAppCta source =
-    { name = "open-app-cta", details = [ ( "source", source ) ], enabled = True }
+openSharing : Maybe { e | project : { p | organization : Maybe { o | id : OrganizationId }, id : ProjectId } } -> TrackEvent
+openSharing erd =
+    createEvent "open-sharing" [] (erd |> Maybe.map .project)
 
 
-openProjectUploadDialog : TrackEvent
-openProjectUploadDialog =
-    { name = "open-project-upload-dialog", details = [], enabled = True }
+openSettings : Maybe { e | project : { p | organization : Maybe { o | id : OrganizationId }, id : ProjectId } } -> TrackEvent
+openSettings erd =
+    createEvent "open-settings" [] (erd |> Maybe.map .project)
 
 
-openSharing : TrackEvent
-openSharing =
-    { name = "open-sharing", details = [], enabled = True }
+openHelp : Maybe { e | project : { p | organization : Maybe { o | id : OrganizationId }, id : ProjectId } } -> TrackEvent
+openHelp erd =
+    createEvent "open-help" [] (erd |> Maybe.map .project)
 
 
-openSettings : TrackEvent
-openSettings =
-    { name = "open-settings", details = [], enabled = True }
+openTableDropdown : { p | organization : Maybe { o | id : OrganizationId }, id : ProjectId } -> TrackClick
+openTableDropdown project =
+    createClick "open-table-dropdown" [] (Just project)
 
 
-openHelp : TrackEvent
-openHelp =
-    { name = "open-help", details = [], enabled = True }
+showTableWithForeignKey : { p | organization : Maybe { o | id : OrganizationId }, id : ProjectId } -> TrackClick
+showTableWithForeignKey project =
+    createClick "show-table-with-foreign-key" [] (Just project)
```

Comment:
same here... Maybe have this as a one-time onboarding achievement more than a forever tracking event.